src/Altinn.App.PlatformServices/Configuration/GeneralSettings.cs

```diff
@@ -22,6 +22,16 @@ public class GeneralSettings
         /// </summary>
         public string FixedValidationPrefix { get; set; }
```

Comment:
Does FIXED == ERROR? Not that I mean we should change, just curious:)

Reply:
No. Default severity is error, so no need to prefix that.

Fixed just indicates that a field has been corrected. This could be all kinds.